app/src/main/java/org/oppia/app/home/recentlyplayed/RecentlyPlayedFragment.kt

```diff
@@ -15,7 +15,7 @@ private const val RECENTLY_PLAYED_FRAGMENT_INTERNAL_PROFILE_ID_KEY =
 /** Fragment that contains all recently played stories. */
 class RecentlyPlayedFragment : InjectableFragment(), OngoingStoryClickListener {
   companion object {
-    const val TAG_RECENTLY_PLAYED_FRAGMENT = "TAG_RECENTLY_PLAYED_FRAGMENT"
+    const val TAG_RECENTLY_PLAYED_FRAGMENT = "RecentlyPlayedFragment.tag"
```

Comment:
Please take a look at this comment:
https://github.com/oppia/oppia-android/pull/1717#discussion_r478456256

As per this comment, you should rename like this:
`internal const val TAG = "RecentlyPlayedFragment.tag`

Reply:
done